packages/model-viewer/src/model-viewer-base.ts

```diff
@@ -128,7 +128,7 @@ export default class ModelViewerElementBase extends UpdatingElement {
   }
 
   get[$renderer]() {
-    return Renderer.singleton;
+    return this[$scene].background ? Renderer.singleton : Renderer.singletonWithTransparency;
```

Comment:
It seems like we should only use the transparent renderer if the background color is, in fact, `"transparent"`. One way to accomplish this would be to specialize the `[$renderer]` accessor for the `EnvironmentMixin` implementation, so that it overrides the default ("always use opaque renderer") with a version that returns a renderer based on the value of the `backgroundColor` property.